src/OrchardCore.Modules/OrchardCore.Shortcodes/Startup.cs

```diff
@@ -68,6 +70,10 @@ public ShortcodeTemplatesStartup(IOptions<AdminOptions> adminOptions)
 
         public override void ConfigureServices(IServiceCollection services)
         {
+            services.AddTransient<IDeploymentSource, AllShortcodeTemplatesDeploymentSource>();
```

Comment:
Should be registered in a different startup, using `RequiresFeatures("OrchardCore.Deployment", "OrchardCore.Shortcodes.Templates")`